models/public/hrnet-w32-human-pose-estimation/hrnet-w32-human-pose-estimation.md

```diff
@@ -0,0 +1,85 @@
+# hrnet-w32-human-pose-estimation
+
+## Use Case and High-Level Description
+
+The `HRNet-W32` model is one of the [HigherHRNet](https://arxiv.org/pdf/1908.10357).
+`HigherHRNet` is a novel bottom-up human pose
```

Comment:
here you speak about HigherHRNet, while you name model without `higher` prefix in demo and models, possibly this prefix should be included into name?

Reply:
Add `higher` prefix